src/pages/frameworks.mdx

```diff
@@ -53,6 +53,144 @@ For building mobile (iOS and Android) applications.
 
 You can also use a framework like [Expo](https://www.npmjs.com/package/expo) to build an application that will run natively on Android, iOS, AND modern web browsers using the same codebase. We plan on building this section out a little more in the future, providing learning resources and common tips for React Native projects. Have ideas? Help us out and [pitch them](https://docs.google.com/document/d/10n5LQH_8VZFMzngMWhY74WrvzqFKF3v-9iUPpiMxMDQ/edit#) in our git repository.
 
+### **Best Practices**
+
+### Responsiveness
+When it comes to responsiveness, you can use `react-native-normalize` to calculate your responsive units. Example:
```

Comment:
I looked into this a little, and it appears to have bad support for android according to the PR’s in the repo, https://www.npmjs.com/package/react-native-normalize and also the weekly downloads < 1000, I don't know if there's enough popularity behind this to recommend it. Is there an article or some documented reason to use this approach? Or maybe there's just a different library that does the same thing and is the popular version?